hack/update-webhook-configurations.yq

```diff
@@ -0,0 +1,8 @@
+with(.metadata;
```

Comment:
Can you add a commnet explaining whats going on here

Reply:
Done.